chore(blog): Summer of Privacy announcement (#20)
blindnet-io/blindnet.dev: blog/2022-06-21-summer-of-privacy.md

And we especially want to avoid making wrong assumption about what **you**, the developers, need.

We especially want you to tell us what you'd expect from such ambitious privacy dev tools.

What problems are you facing when embracing Privacy-by-Design?\
What are you worst pain points when enforcing privacy regulations (e.g. GDPR)?\
What do you need from an app to be sure to sufficiently trust it?

Here, we already give you several opportunities to do that:

1. our recently opened [Slack Workspace](https://join.slack.com/t/blindnet/shared_invite/zt-1arqlhqt3-A8dPYXLbrnqz1ZKsz6ItOg), to chat with the blindnet community and teams;
1. an open [DevRel Management](https://github.com/blindnet-io/product-management) repository, when you can just [make simple informal requests](https://github.com/blindnet-io/devrel-management/issues/new?assignees=noelmace&labels=request%2Ctriage&template=request.yml&title=%5BRequest%5D%3A+);
1. an open [Product Management](https://github.com/blindnet-io/product-management/) repository, where you can track, read, and contribute to all parts of our product development process, from simple bug fixes to complexe architecture reference documents.

> If those still seem to complexe or frightening to you, just email [hello@blindnet.io](mailto:hello@blindnet.io), and we'll make sure to give you an answer right away.

Reviewer: ```suggestion
If those still seem too complex or frightening to you, just email [hello@blindnet.io](mailto:hello@blindnet.io), and we'll make sure to give you an answer right away.
```

> ## Helping Everyone to Better Understand Privacy

Reviewer 2: ```suggestion
## Helping Everyone Better Understand Privacy
```

While getting on with this huge project, we'll also make sure to keep you posted in any possible way.

So first, you should definitly follow us on [Twitter](https://twitter.com/blindnet_io), as this will be the first place where we'll post short and constant updates on all technical matters.

Our new engineering blog, on both [blindnet.dev](https://blindnet.dev/blog) and [dev.to](https://dev.to/blindnet), will then be a good place to share more in-depth explainers and update about our devkit and privacy in general.
Where are open to any suggestion of topic to address, and even guest blog posts.
So make sure to contact us whenever privacy inspire you.

> Finally, we'll soon open a new Twitch Channel and Youtube channel, with regular programs and  improvised streaming.

Reviewer: ```suggestion
Finally, we'll soon open a new Twitch Channel and Youtube channel, with regular programs and improvised streaming.
```
